### DISCUSSION TITLE
HOPPE eHandle ConnectSense

### DISCUSSION THREAD

**Discussion by lieblinger** [zwave-js/zwave-js#6534] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
There's no output on node-zwave-js for the handle position state (open/close). It only publishes the state of the window sash (open/close).

What did you expect to happen instead?
According to the the manual it should expose the following:
![Bildschirmfoto vom 2023-12-01 21-03-08](https://github.com/zwave-js/node-zwave-js/assets/10740088/7c3c3fc2-10bb-484d-af85-71035cd813dd)

Shouldn't be there a sensor for "Window handle in closed position (0x06 / 0x01)" and "Window handle in open position (0x06 / 0x02)" ?

### Device information

Manufacturer: Hoppe
Model name: E0400Z-EF
Node ID: 016 and 063

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[log2.txt](https://github.com/zwave-js/node-zwave-js/files/13531666/log2.txt)

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment, marked as answer]
0x06/0x01, 0x06/0x02 and 0x06/0x07 are notification **events**, not **variables**. By definition, events are stateless and only carry a meaning the instant they are received. They mean "manual lock operation", "manual unlock operation" and "manual not fully locked" respectively.

It seems that HOPPE have interpreted the Z-Wave specifications in an unfortunate way here, because there are actual states for handle open (0x06/0x18) and handle closed (0x06/0x19) in the specifications, which would do what you'd expect.
